front/app/containers/Admin/pagesAndMenu/EditHomepage/HeroBanner/index.tsx

```diff
@@ -32,9 +32,19 @@ const EditHomepageHeroBannerForm = ({
 }: InjectedIntlProps) => {
   const [isLoading, setIsLoading] = useState(false);
   const [formStatus, setFormStatus] = useState<ISubmitState>('disabled');
+  const [localSettings, setLocalSettings] =
+    useState<IHomepageSettingsAttributes | null>(null);
```

Comment:
I don't think these types will work as the values from the generic form are using the generic property names ("`banner_header_multiloc` instead of `banner_signed_out_header_multiloc`), so the call from the generic form will be `changeValue('banner_header_multiloc`, value)` which isn't a key on HomepageSettingsAttributes